lib/IRGen/GenProto.cpp

```diff
@@ -1276,7 +1280,18 @@ class AccessorConformanceInfo : public ConformanceInfo {
       }
       return *Fulfillments;
     }
-  };
+
+    void bindArchetypeAccessPathsInConformance(IRGenFunction &IGF) {
+      auto TheDecl = Conformance.getDeclContext();
```

Comment:
The standard abbreviation for a DeclContext is DC.  Please don't call it a Decl; that's actively somewhat misleading.